lib/android_lint/plugin.rb

```diff
@@ -131,10 +131,8 @@ def message_for_issues(issues)
     def parse_results(results, heading)
       target_files = (git.modified_files - git.deleted_files) + git.added_files
       dir = "#{Dir.pwd}/"
-      message = "#### #{heading} (#{results.count})\n\n"
-
-      message << "| File | Line | Reason |\n"
-      message << "| ---- | ---- | ------ |\n"
+      count = 0;
```

Comment:
It seems to me that this variable isn`t incremented anywhere.

Reply:
Updated. Sorry for the missing code.